gcip/addons/container/jobs/crane.py

```diff
@@ -0,0 +1,50 @@
+from typing import Union, Optional
+
+from gcip.core.job import Job
+from gcip.core.image import Image
+from gcip.addons.container.config import DockerClientConfig
+from gcip.addons.container.images import PredefinedImages
+
+
+def copy(
+    src: str,
+    dst: str,
+    *,
+    docker_client_config: Optional[DockerClientConfig] = None,
+    crane_image: Optional[Union[Image, str]] = None,
+) -> Job:
+    """
+    Creates a job to copy container images with `crane`.
+    See [`crane`](https://github.com/google/go-containerregistry/tree/main/cmd/crane)
+
+    Copying an image is usfull, if you want to have container images as close as possible
+    to your cluster or servers.
+
+    Args:
+        src (str): Registry URL to copy container image from.
+        dst (str): Registry URL to copy container image to.
+        docker_client_config (Optional[DockerClientConfig], optional): Creates the Docker configuration file base on objects settings,
+            used by crane to authenticate against given registries. Defaults to None.
+        crane_image (Optional[Union[Image, str]], optional): Container image which contains `crane` command.
+            Defaults to PredefindedImages.CRANE.
+
+    Returns:
+        Job: Returns a `gcip.Job`, with neccessary configuration to copy a container image between images.
+    """
+    if not crane_image:
+        crane_image = PredefinedImages.CRANE
+
+    job = Job(
+        script=[
+            "date",
```

Comment:
Das hatte ich bei dem Kaniko Job nur eingefügt, weil ich den Job ganz am Anfang der Funktion ohne vorhandenes Script angelegt habe. Da ein Job aber immer ein Script haben muss, hatte ich dort einfach `date` eingefügt. Hat keinen funktionalen Wert. Und date ist auch in der Gitlab gui ersichtlich.

```suggestion
```